src/aero_kernel/src/fs/inode.rs

```diff
@@ -309,6 +333,30 @@ impl DirEntry {
         })
     }
 
+    pub fn from_inode_custom(
```

Comment:
Maybe rename the function to something else. Since specifically in this function we create a local socket inode.

Reply:
How does `from_socket_inode` sound?

Reply:
👍🏼

Reply:
Renamed in 1b082c3ac67474a99a180c0fb329b1d7a8da2872